src/payment/order-details/OrderDetails.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FormattedMessage, injectIntl, intlShape } from '@edx/frontend-i18n';
+
+import messages from './messages';
+import { orderDetailsMapStateToProps } from './data/selectors';
+
+class OrderDetails extends Component {
+  renderSimpleMessage() {
+    return (
+      <p>
+        {this.props.intl.formatMessage(messages[`payment.order.details.course.${this.props.messageType}`])}
+      </p>
+    );
+  }
+
+  renderEnrollmentCodeMessage() {
+    return (
+      <React.Fragment>
+        <p>By purchasing, you and your organization agree to the following terms:</p>
```

Comment:
Do we need to i18n this?

Reply:
Oh junk!  Wow!  I totally forgot.  Of course it does. :open_mouth: